stan/math/prim/scal/meta/scalar_seq_view.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef STAN_MATH_PRIM_SCAL_META_SCALAR_SEQ_VIEW_HPP
+#define STAN_MATH_PRIM_SCAL_META_SCALAR_SEQ_VIEW_HPP
+
+#include <stan/math/prim/scal/meta/scalar_type.hpp>
+
+namespace stan {
+
+  template <typename C, typename T = typename scalar_type<C>::type>
```

Comment:
Add documentation for the class. See https://github.com/stan-dev/stan/wiki/How-to-Write-Doxygen-Doc-Comments